### PR TITLE
Fix CronJob return type

### PR DESCRIPTION
**Pull request by VandeurenGlenn**

### CODE REVIEW

**Comment by vdegenne** [comment]
lgtm

File: packages/server/src/index.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ async function loadData() {
 await loadData();
 
 // fetch remotely every hour
-const job = new CronJob('0 * * * *', updateCacheWithRemote);
+const job = new CronJob('0 * * * *', updateCacheWithRemote as unknown as CronCommand<any>);
```
Comment (vdegenne) [inline packages/server/src/index.ts:22]: not looking good but that'll do lol

**Review by vdegenne — COMMENTED** [review]
LGTM

File: packages/server/src/index.ts (hunk 1 above)
Comment (VandeurenGlenn) [reply packages/server/src/index.ts:22]: Better then ignore I guess

**Review by vdegenne — APPROVED** [review]
LGTM!!!!!!!!!!!

**Pull request merged**